### PR TITLE
Add License file to repo

### PR DESCRIPTION
**Pull request by jeremy-moffitt**
added Apache license to repo

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,176 @@
+pache License
```
Comment (nicolasbock) [inline LICENSE]: Copy and paste issue?

Comment (toabctl) [inline LICENSE]: needs to be fixed

**Comment by jeremy-moffitt** [comment]
Yes, copy/paste error, will fix that up shortly

**Comment by jeremy-moffitt** [comment]
@dirkmueller  @toabctl  please re-review with the latest patch set, thanks!

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+
```
Comment (toabctl) [inline LICENSE]: One empty line should be removed. anyway - merging ....

**Pull request merged**